### DISCUSSION TITLE
Seq on ECS Fargate with EFS storage

### DISCUSSION THREAD

**Discussion by rofr** [datalust/seq-tickets#1894] [Help] [answered]
Hi all,

I'm going to do a test on AWS using ECS Fargate (serverless docker containers) as I want to avoid managing EC2 instances. The only storage option is EFS, network attached storage. I understand that block storage is preferred but assume that EFS should work but perhaps with worse performance. Or are there other issues than performance?

**Comment by andymac4182** [comment, marked as answer]
Personally I have used fargate and efs multiple times and no issues other than under heavy load the perf drops a bit but personally I never noticed any problems

**Reply by rofr** [reply]
Thanks, that's good to know. I decided to play it safe with an EC2 instance and an attached EBS volume.

**Comment by liammclennan** [comment]
I can't comment on EFS specifically, but network file systems generally don't provide the consistency necessary for a database to be reliable. Every database I've checked (sqlite, postgresql, sql server) warns against network storage for the same reason. Expect occasional corruption under heavy load.

**Reply by rofr** [reply]
Thanks. I ended up going with the EC2 instance + EBS approach anyway.
